apis/supernova/src/hooks.js

```diff
@@ -772,6 +772,31 @@ export function useDeviceType() {
   return useInternalContext('deviceType');
 }
 
+/**
+ * Gets the navigation api to control sheet navigation
+ * @entry
+ * @returns {Navigation} navigation api.
+ * @example
+ * // Render a sheet
+ * const nebbie = embed(app);
+ * const sheetRender = await nebbie.render({
+ *   element: sheetElement
+ *   id: 'sheetId',
+ * });
+ *
+ * @example
+ * // Use a navigation object to control sheet navigation
+ * const plugins = usePlugins();
+ * const navigationRender = await nebbie.render({
+ *   element: navigationElement
+ *   id: 'navigationId',
+ *   navigation: sheetRender.navigation
+ * });
```

Comment:
maybe wrong place for this example?
it if for how to use navigation with .render function not how to use useNavigation

Reply:
Indeed it is a wrong place. Updated!